README.md

```diff
@@ -18,24 +18,24 @@ git submodule update --init --recursive
 ```
 
 ## Important branches to know
-master            --> Development is done continuously on this branch  
-release           --> Fully tested released source code  
-release-candidate --> Preview of upcoming release  
-feature/*         --> Alpha/beta of an upcoming feature  
```

Comment:
Don't remove trailing line endings in .md files. They're needed to display text in new line